specutils/io/default_loaders/manga.py

```diff
@@ -0,0 +1,141 @@
+
+import astropy.units as u
+from astropy.io import fits
+from astropy.nddata import InverseVariance
+
+from ...spectra import Spectrum1D
+from ..registers import data_loader
+from ..parsing_utils import read_fileobj_or_hdulist
+
+
+__all__ = ["identify_manga_cube", "identify_manga_rss", "manga_cube_loader", "manga_rss_loader"]
+
+
+def identify_manga_cube(origin, *args, **kwargs):
+    """
+    Check whether the given file is a MaNGA CUBE.
+    """
+
+    with read_fileobj_or_hdulist(*args, **kwargs) as hdulist:
+        return (hdulist[0].header["TELESCOP"] == "SDSS 2.5-M" and "FLUX" in hdulist
+                and hdulist[1].header['INSTRUME'] == 'MaNGA'
+                and hdulist[1].header["NAXIS"] == 3)
+
+
+def identify_manga_rss(origin, *args, **kwargs):
+    """
+    Check whether the given file is a MaNGA RSS.
+    """
+
+    with read_fileobj_or_hdulist(*args, **kwargs) as hdulist:
+        return (hdulist[0].header["TELESCOP"] == "SDSS 2.5-M" and "FLUX" in hdulist
+                and hdulist[1].header['INSTRUME'] == 'MaNGA'
+                and hdulist[1].header["NAXIS"] == 2)
+
+
+@data_loader("MaNGA cube", identifier=identify_manga_cube, dtype=Spectrum1D,
+             extensions=['fits'])
+def manga_cube_loader(file_obj, **kwargs):
+    """
+    Loader for MaNGA 3D rectified spectral data in FITS format.
+
+    Parameters
+    ----------
+    file_obj: str, file-like, or HDUList
+          FITS file name, object (provided from name by Astropy I/O Registry),
+          or HDUList (as resulting from astropy.io.fits.open()).
+
+    Returns
+    -------
+    Spectrum1D
+        The spectrum contained in the file.
+    """
+
+    if isinstance(file_obj, fits.hdu.hdulist.HDUList):
+        hdulist = file_obj
+    else:
+        hdulist = fits.open(file_obj, **kwargs)
+
+    spaxel = u.Unit('spaxel', represents=u.pixel, doc='0.5" spatial pixel', parse_strict='silent')
```

Comment:
This is actually an interesting and good use of units.  I wonder if this should be moved to the public API so that users can do e.g. `specutils.<something>.spaxel`? 

(this could be done in a folllow-on PR if you don't want to deal with it right now though - I can make an issue to deal with later if that's your preference @havok2063)